### ISSUE TITLE
Automatic translations for the app descriptions

### ISSUE THREAD

**Issue by srueegger** [ubuntu/app-center#948]
On a localized system, the Snap Store displays app descriptions in English.

My suggestion would be that if an app doesn't have a localized description, an auto-translated version (with appropriate notice) is displayed.

For translation one could work with the "LibreTranslate" project or "The Bergamot Project Consortium".

![translate-snap-2](https://user-images.githubusercontent.com/11472914/216539388-343e473e-040f-4cf9-a610-541044acf69a.png)

**Comment by Feichtmeier** [comment]
hi @srueegger thanks for the idea

I would prefer if we could do this in the snap backend though

@local-optimum  do you think this is doable or rather not?

**Comment by local-optimum** [comment]
I agree it would be better to support this in the snap backend to ensure we've hit the quality bar in translations and don't accidentally misrepresent an app description, let me raise it with the snapstore team.

**Comment by d-loose** [comment]
@local-optimum are there any news on this?

**Comment by ogres** [comment]
We cannot do this unfortunately, as it might lead to unexpected issues if translation _slightly_ changes the context of the text.
